CHANGELOG.md

```diff
@@ -64,6 +64,7 @@
 - Added support for `DataFrame.align` and `Series.align` for `axis=1` and `axis=None`.
 - Added support fot `pd.json_normalize`.
 - Added support for `GroupBy.pct_change` with `axis=0`, `freq=None`, and `limit=None`.
+- Added support for `DataFrameGroupBy.__iter__` and `SeriesGroupBy.__iter__`.
```

Comment:
Merge artifact?

Reply:
For me, this line doesn't show up in the diff against main.